.stylelintrc.json

```diff
@@ -18,5 +18,5 @@
     },
```

Comment:
### Your PR Title and Branch name
![Screenshot from 2022-09-20 16-00-33](https://user-images.githubusercontent.com/93039359/191242216-6df880e6-9413-4ce5-a0e3-2d6b718302fb.png)

- [ ] Your pull request title is not descriptive. You are using **Updated .stylelintrc** you can use **Setup Hello Microverse Project** which is descriptive and tellling your viewers what you are doing in this pull request :blush:
- [ ] Your branch name is not descriptive enough. You can use **feature/setup-hello-microverse** which is very descriptive and telling your viewers what are you doing in this branch :+1: